dts/xtensa/espressif/esp32/esp32_common.dtsi

```diff
@@ -94,9 +94,16 @@
 	};
 
 	soc {
-		sram0: memory@3ffb0000 {
```

Comment:
I think you should state the `sram0` here with Ibus addresses and a size of `192kB` just for a sake of having those values in dts only.